### PR TITLE
Improved controllers descriptions.

### PR DESCRIPTION
**Pull request by MateuszSz93**

### CODE REVIEW

File: src/main/java/com/intive/patronage/toz/howtohelp/DonateInfoController.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 import javax.validation.Valid;
 
-@Api("Information on how to donate.")
+@Api(tags = "Donate info", description = "Information about how to donate.")
```
Comment (mlotysz) [inline src/main/java/com/intive/patronage/toz/howtohelp/DonateInfoController.java]: Missing word: `Manage...`.

File: src/main/java/com/intive/patronage/toz/organization/OrganizationInfoController.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 import javax.validation.Valid;
 import java.net.URI;
 
-@Api(tags = "Organization info")
+@Api(tags = "Organization info", description = "Information about organization.")
```
Comment (mlotysz) [inline src/main/java/com/intive/patronage/toz/organization/OrganizationInfoController.java]: I'd add `manage` somewhere - we provide CRUD operations...

File: src/main/java/com/intive/patronage/toz/tokens/TokensController.java [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 import javax.validation.Valid;
 
-@Api(tags = "Tokens")
+@Api(tags = "Tokens", description = "Tokens operations (operations related to authentication.)")
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/tokens/TokensController.java]: Dot should be outside brackets.

File: src/main/java/com/intive/patronage/toz/passwords/PasswordsController.java [hunk 4]
```diff
@@ -36,7 +36,7 @@
 import static com.intive.patronage.toz.config.ApiUrl.PASSWORDS_RESET_PATH;
 import static com.intive.patronage.toz.config.ApiUrl.PASSWORDS_RESET_SEND_TOKEN_PATH;
 
-@Api(tags = "Passwords", description = "Changing passwords.")
+@Api(tags = "Passwords", description = "Passwords operations (change and reset password).")
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/passwords/PasswordsController.java]: In my opinion, additional word `password` between brackets is unnecessary.

**Pull request merged**